pkg/api/meta/meta.go

```diff
@@ -506,6 +608,15 @@ func extractFromObjectMeta(v reflect.Value, a *genericAccessor) error {
 	if err := runtime.FieldPtr(v, "Annotations", &a.annotations); err != nil {
 		return err
 	}
+	ownerReferences := v.FieldByName("OwnerReferences")
```

Comment:
@lavalamp this is still WIP, but would be great if you can take a look if this is the way we want to go.